Spec/Auth.swift

```diff
@@ -4030,24 +3982,45 @@ override class var defaultTestSuite : XCTestSuite {
                     }
                 }
             }
-        }
+enum TestCase_ReusableTestsTestTokenRequestFromJson {
+case accepts_a_string__which_should_be_interpreted_as_JSON
+case accepts_a_NSDictionary
+}
+
+                func reusableTestsTestTokenRequestFromJson(_ json: String, testCase: TestCase_ReusableTestsTestTokenRequestFromJson, context: (beforeEach: (() -> ())?, afterEach: (() -> ())?), check: @escaping (_ request: ARTTokenRequest) -> Void) {
+                    func test__accepts_a_string__which_should_be_interpreted_as_JSON() {
+context.beforeEach?()
 
-        describe("TokenRequest") {
-            // TE6
-            describe("fromJson") {
-                func reusableTestsTestTokenRequestFromJson(_ json: String, check: @escaping (_ request: ARTTokenRequest) -> Void) {
-                    it("accepts a string, which should be interpreted as JSON") {
                         check(try! ARTTokenRequest.fromJson(json as ARTJsonCompatible))
+context.afterEach?()
+
                     }
 
-                    it("accepts a NSDictionary") {
+                    func test__accepts_a_NSDictionary() {
+context.beforeEach?()
+
                         let data = json.data(using: String.Encoding.utf8)!
                         let dict = try! JSONSerialization.jsonObject(with: data, options: .mutableLeaves) as! NSDictionary
                         check(try! ARTTokenRequest.fromJson(dict))
+context.afterEach?()
+
                     }
+
+switch testCase  {
+case .accepts_a_string__which_should_be_interpreted_as_JSON:
+    test__accepts_a_string__which_should_be_interpreted_as_JSON()
+case .accepts_a_NSDictionary:
+    test__accepts_a_NSDictionary()
+}
+
                 }
 
-                context("with TTL") {
+        
+            // TE6
```

Comment:
TODO this should be above `reusableTestsTestTokenRequestFromJson`

Reply:
(Observation, doesn't block review)

The migrator has moved put this comment in the wrong place for some reason. It should be above  the declaration of `reusableTestsTestTokenRequestFromJson`. I'll see if I can figure out why, but if it's a huge amount of work then I'll just fix this manually (it seems to be the only such affected comment).